next-frontend/src/components/ui/errored.tsx

```diff
@@ -1,13 +1,65 @@
-import { Alert } from "@chakra-ui/react";
+import {
+  Accordion,
+  Alert,
+  CodeBlock,
+  Container,
+  Float,
+  IconButton,
+} from "@chakra-ui/react";
+import { TFunction } from "i18next";
 
-export default function Errored({ error }: { error: string }) {
+export default function Errored({
+  t,
+  response,
+}: {
+  response: Response;
+  t: TFunction;
+}) {
   return (
-    <Alert.Root>
-      <Alert.Indicator />
-      <Alert.Content>
-        <Alert.Title>Error fetching Data</Alert.Title>
-        <Alert.Description>{error}</Alert.Description>
-      </Alert.Content>
-    </Alert.Root>
+    <Container>
+      <Alert.Root status="error">
+        <Alert.Indicator />
+        <Alert.Content>
+          <Alert.Title>{t("errors.next_frontend.title")}</Alert.Title>
+          <Alert.Description>
+            {t("errors.next_frontend.description")}
+            <Accordion.Root collapsible>
+              <Accordion.Item value="details">
+                <Accordion.ItemTrigger>
+                  {t("errors.next_frontend.details")}
+
+                  <Accordion.ItemIndicator />
+                </Accordion.ItemTrigger>
+                <Accordion.ItemContent>
+                  <Accordion.ItemBody>
+                    <CodeBlock.Root
+                      meta={{ wordWrap: true }}
+                      code={JSON.stringify({
```

Comment:
For now, just `JSON.stringify(yourStuff, null, 2)` should do I think. As a compromise.